NEMP/mods.json

```diff
@@ -4101,24 +4227,6 @@
             "regex" : "BiomeTweaker-(?P<mc>.+?) (?P<version>.+)"
         }
     },
-    "Caterpillar": {
-        "function": "CheckCurse",
-        "active": true,
-        "curse": {
-            "regex": "Caterpillar-(?P<version>.+)\\.jar",
-            "id": "237628",
-            "name": "caterpillar"
-        }
-    },
-    "UnLogicII": {
```

Comment:
Pyker: UnLogic II is one of my mods, and has been replaced by Fire's Random Things. There is zero chance of it being updated by me, and nobody else should have any reason to update it.